packages/ckeditor5-ui/docs/features/menubar.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Menu bar
+category: features-toolbar
+meta-title: Menu bar | CKEditor 5 Documentation
+order: 30
+---
+{@snippet features/build-menubar-source}
+
+# Menu bar
+
+The toolbar is the most basic user interface element of CKEditor&nbsp;5 that gives you convenient access to all its features. It has buttons and dropdowns that you can use to format, manage, insert, and change elements of your content.
+
+## Demo
+
+Below is a sample menu bar with a basic set of features. Menu items can be easily added or removed. Read on to learn how to do that.
```

Comment:
```suggestion
Below is a demo presenting a sample menu bar for an editor with a basic set of features. Menu items can be easily added or removed. Read on to learn how to do that.
```

Reply:
Not sure if this is a good place to mention that the items can be added or removed here. OTOH, we should mention (here, or above) that the menu bar is automatically filled with items for all features loaded in the editor. So maybe something like (in short): 1. mention that menu is automatically filled; 2. mention that you can remove and reorganize the menu bar structure; 3. mention that you can add menu bar items for custom features.

I think I'd add this all below the demo, so that the demo is higher on the page.

Reply:
In commit.

Reply:
> we should mention (here, or above) that the menu bar is automatically filled with items for all features loaded in the editor

This is kind of mentioned in the section below when discussing default structure. I think I'd still mention it somewhere at the beginning and directly.

Reply:
Is it filled with ALL items available, or within the preset only?

Reply:
All features **loaded in the editor**, so with all items for the available plugins (expect maybe a few which we didn't have time to integrate or that do not make sense in the menu bar).